src/components/molecules/CashAcclerationDashboardCardItem/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box } from "@mui/material";
+import theme from "../../../utils/theme";
+import IconFromSvg from "../../atoms/IconFromSvg";
+import TypographyWithInfoIcon from "../TypographyWithInfoIcon";
+import Typography from "../../atoms/Typography";
+
+export interface IDashboardCardItemProps {
+    icon: string;
+    infoText: string;
+    title: string
+};
+
+const DashboardCardItem = ({ icon, infoText, title }: IDashboardCardItemProps) => {
```

Comment:
you have have the file  name also as  DashboardCardItem  as molecules are reusable in other parts of code. So file name must be generic.